client/src/partials/reports_proposed/debitor_group_report/debitor_group_report.js

```diff
@@ -0,0 +1,74 @@
+var DebtorGroupReportController = function ($http, $translate, validate, reportConfigService, messenger) {
+
+  var vm = this,
+  generatedDocumentPath = null,
+  serverUtilityPath = '/report/build/debitor_group_report',
+  configuration = reportConfigService.configuration,
+  dependencies = {};
+
+  dependencies.debitorGroup = {
+      query : {
+        identifier : 'uuid',
+        tables : {
+          'debitor_group' : {
+            columns : ['uuid', 'name', 'account_id', 'is_convention']
+          }
+        },
+        where : ['debitor_group.is_convention=1']
+      }
+  };
+
+  vm.generate_doc = $translate.instant('DEBITOR_GROUP_REPORT.GENERATE_DOC');
```

Comment:
$translate service works fine and changes when the translation changes.
I avoid to use it on the view because, it will make the html page unreadable (putting conditional structure on a html page is not catholic).